diff --git a/cmsplugin_filer_folder/cms_plugins.py b/cmsplugin_filer_folder/cms_plugins.py
index 36bd802..3f29fc4 100644
--- a/cmsplugin_filer_folder/cms_plugins.py
+++ b/cmsplugin_filer_folder/cms_plugins.py
@@ -5,9 +5,13 @@ from django.utils.translation import ugettext_lazy as _
 from . import models
 from .conf import settings
 
+from filer.models.abstract import BaseImage
 from filer.models.filemodels import File
 from filer.models.foldermodels import Folder
-from filer.models.abstract import BaseImage
+from filer.settings import FILER_IMAGE_MODEL
+from filer.utils.loader import load_model
+
+Image = load_model(FILER_IMAGE_MODEL)
 
 
 class FilerFolderPlugin(CMSPluginBase):
@@ -51,7 +55,7 @@ class FilerFolderPlugin(CMSPluginBase):
             folder_folders = self.get_children(instance.folder)
         else:
             folder_files = File.objects.none()
-            folder_images = BaseImage.objects.none()
+            folder_images = Image.objects.none()
             folder_folders = Folder.objects.none()
 
         context.update({
-- 
2.25.1
