--- cmsplugin_filer_folder/cms_plugins.py.orig	2018-09-27 00:14:04.900014843 +0200
+++ cmsplugin_filer_folder/cms_plugins.py	2018-09-27 00:15:10.389939948 +0200
@@ -51,7 +51,7 @@ class FilerFolderPlugin(CMSPluginBase):
             folder_folders = self.get_children(instance.folder)
         else:
             folder_files = File.objects.none()
-            folder_images = BaseImage.objects.none()
+            folder_images = File.objects.none()
             folder_folders = Folder.objects.none()
 
         context.update({
