--- filer/admin/imageadmin.py.orig	2018-09-24 06:38:40.000000000 +0000
+++ filer/admin/imageadmin.py	2018-09-26 21:41:24.966895111 +0000
@@ -35,6 +35,7 @@ class ImageAdminForm(forms.ModelForm):
     def _set_previous_subject_location(self, cleaned_data):
         subject_location = self.instance.subject_location
         cleaned_data['subject_location'] = subject_location
+        self.data = self.data.copy()
         self.data['subject_location'] = subject_location
 
     def clean_subject_location(self):
